stdpopsim/cli.py

```diff
@@ -138,100 +138,137 @@ def summarise_usage():
         user_time, sys_time, max_rss))
 
 
-def add_model_runner(top_parser, model):
-    """
-    Adds CLI options and registers the runner callback for the specified model.
-    """
-    parser = top_parser.add_parser(
-        model.name,
-        description=model.description,
-        help=model.short_description)
-    add_output_argument(parser)
-    sample_arg_names = []
-    if len(model.populations) == 1:
-        assert model.populations[0].allow_samples
-        help_text = f"The number of samples"
-        parser.add_argument(f"--num-samples", help=help_text, default=0, type=int)
-        sample_arg_names.append("num_samples")
-    else:
+def add_simulate_species_parser(parser, species):
+    header = (
+        f"Run simulations for {species.name} using up-to-date genome information, "
+        "genetic maps and simulation models from the literature."
+    )
+
+    # TODO perhaps it would be better to have a model help text option rather than
+    # writing all this out every time. It's going to be a lot of text, and might
+    # distract from the important help stuff.
+    models_text = "\nSimulation models\n\n"
+    indent = " " * 4
+    wrapper = textwrap.TextWrapper(initial_indent=indent, subsequent_indent=indent)
+    for model in species.models:
+        models_text += f"{model.id}: {model.name}\n"
+        models_text += wrapper.fill(textwrap.dedent(model.description))
+        models_text += "\n\n"
+
+        models_text += indent + "Populations:\n"
+
         for population in model.populations:
             if population.allow_samples:
-                name = population.name
-                help_text = f"The number of samples to take from the {name} population"
-                parser.add_argument(f"--num-{name}", help=help_text, default=0, type=int)
-                sample_arg_names.append("num_{}".format(name))
+                models_text += indent * 2
+                models_text += f"{population.name}: {population.description}\n"
+        models_text += "\n"
+
+    description_text = textwrap.fill(header) + "\n" + models_text
+
+    species_parser = parser.add_parser(
+        f"sim-{species.id}",
```

Comment:
adding the `sim-` species id means a bit of extra typing. can we ditch the `sim-` prefix here and instead have the user just start with the species name?

Reply:
I struggled with this a bit, and decided on the prefix after thinking a bit about the cache management commands we'll want as well (say, ``download-genetic-maps``, ``cache-ls``, ``cache-rm``). It seems nice to distinguish the simulate commands from non-simulate.

We can also add in synonyms for each of the simulate commands like ``sim-homsap`` as ``homsap``, which people can use?

I dunno, I'm happy with either tbh. If you said "I definitely think it's a better idea to drop the prefix", I'd very happily make the change.

Reply:
You're right, let's ditch the prefix. Thanks!

Reply:
Done.